pages/components/SeasonSelector.js

```diff
@@ -1,18 +1,102 @@
 import React, { useState }from 'react';
 
 export default function SeasonSelector({onChange}) {
-    const years = ['Seasons', '2003', '2004', '2005', '2006']
+    const years = [
```

Comment:
Usually, in javascript, the convention is that a variable thats never gonna change (like seasons) are `UPPER_CASE`. So , I'd expect this as `SEASONS`.

Also, IMO, this should be in a separate file called constants.